pkgs/os-specific/linux/vmware/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchFromGitHub, kernel, kmod, gnugrep, vmware-workstation }:
+
+stdenv.mkDerivation rec {
+  pname = "vmware-modules";
+  version = "${vmware-workstation.version}-${kernel.version}";
+
+  src = fetchFromGitHub {
+    owner = "mkubecek";
+    repo = "vmware-host-modules";
+    rev = "w${vmware-workstation.version}-k5.17";
```

Comment:
Should we get 5.17 from the kernel?

Reply:
Not really needed as latest wxxx_k5.17 branch includes all patches that are backwards compatible with previous kernel versions and should compile just fine.